repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -97,6 +111,11 @@ def get_transaction_configuration():
     return transaction_configuration
 
 
+def filter_releases_by_target(releases, target):
+    return [r for r in releases if version.matches_version(
+        ['<= {}.{}'.format(target[0], target[1])], '{}.{}'.format(r[0], r[1]))]
```

Comment:
```suggestion
        ['<= {}.{}'.format(*target)], '{}.{}'.format(*r))]
```

Just FYI - Not necessary to use, but it's shorter